pages/help/services-resources/index.vue

```diff
@@ -48,29 +36,101 @@ if (!data.value.data && !data.value.helpTopicData) {
   throw createError({ statusCode: 404, message: 'Page not found', fatal: true })
 }
 
+// ELASTIC SEARCH INDEX
+// GETS DATA FROM CRAFT
+// CREATES ES INDEX TO BE SEARCHED
+// CHECK THAT NUXT IS RUNNING ON THE SERVER (process.server)
+console.log('DATA-DATA-DATA-DATA' + data)
+if (
+  data.value.externalResource &&
+  data.value.externalResource.length > 0 &&
+  process.server
+) {
+  for (const externalResource of data.value.externalResource) {
+    await $elasticsearchplugin.index(
+      { ...externalResource, serviceOrResourceType: 'external resource' },
+      externalResource.slug
+    )
+  }
+}
+
 const page = ref(data.value.data)
 const helpTopic = ref(data.value.helpTopicData)
 const summaryData = ref(_get(data.value.data, 'entry', {}))
+
+// ENABLE PREVIEW
 watch(data, (newVal, oldVal) => {
   console.log('In watch preview enabled, newVal, oldVal', newVal, oldVal)
   page.value = _get(newVal.data, 'entry', {})
   helpTopic.value = ref(newVal.helpTopicData)
```

Comment:
I noticed we need to make a change here, not related to ES work, it is just a bug
Replace `page.value = _get(newVal.data, 'entry', {})` with `page.value = newVal.data`
Replace `helpTopic.value = ref(newVal.helpTopicData)` with `helpTopic.value = newVal.helpTopicData`